config/tasks/evaluate/test.yaml

```diff
@@ -1,7 +1,5 @@
-# @package _group_
-report_test_metrics: false
 data_loaders:
   test:
     split_key: test_idxs
-    batch_size: 512
-    num_workers: 4
\ No newline at end of file
+    batch_size: 4096
```

Comment:
change in option to be documented?

Reply:
Just a change for convenience when choosing this config file. I don't think we should document every option for all the config files that are supposed to be self-documenting, where users should read the config files they are choosing to use.